### ISSUE TITLE
Add support to upload directly to google docs

### ISSUE THREAD

**Issue by tripodsan** [adobe/helix-importer#38]
For customers that use google drive as content source, it would be useful if they could upload directly to google docs instead of creating the docx first. 

- the importer creates a gdocs json
- the client side has a mechanism to allow the user to login to google
- the client would upload the json via the google doc api

**Comment by kptdobe** [comment]
To be tested but I assume you could mount gdrive locally and write there. This would delegate to gdrive the upload / sync back which can be tricky: during a large import, you need would to make sure the documents are properly uploaded, retry if it fails... makes things just a lot more complicated.

**Comment by tripodsan** [comment]
and what content do you write to your local google docs? I think it's better to upload directly to gdrive during the import.

**Comment by kptdobe** [comment]
I just tested and the auto docx-gdoc conversion does not happen via gdrive. My proposal does not work then.

**Comment by kptdobe** [comment]
Correct. And with a large import, you probably do not want to use this feature: if you are processing 75k urls, the direct upload to gdrive will slow down the process, if process is too fast you might hit the API rate limit, the upload might fail and break the import process...
I personally prefer to import the files on my local file system so that I can upload them manually at the end of the import process. If something goes wrong, I still have the files locally.
